src/main/java/io/quarkus/gizmo2/creator/ops/StringBuilderOps.java

```diff
@@ -0,0 +1,82 @@
+package io.quarkus.gizmo2.creator.ops;
+
+import io.quarkus.gizmo2.Constant;
+import io.quarkus.gizmo2.Expr;
+import io.quarkus.gizmo2.creator.BlockCreator;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * Operations on {@link StringBuilder}.
+ */
+public final class StringBuilderOps extends ObjectOps implements ComparableOps {
+
+    /**
+     * Construct a new instance on an existing {@link StringBuilder}.
+     *
+     * @param bc the block creator (must not be {@code null})
+     * @param obj the receiver string builder (must not be {@code null})
+     */
+    public StringBuilderOps(final BlockCreator bc, final Expr obj) {
+        super(StringBuilder.class, bc, obj);
+    }
+
+    /**
+     * Construct a new instance on a newly created {@link StringBuilder}.
+     * The {@code StringBuilder} instance may be obtained using {@link #getInstance()}.
+     *
+     * @param bc the block creator (must not be {@code null})
+     */
+    public StringBuilderOps(final BlockCreator bc) {
+        super(StringBuilder.class, bc, bc.define("stringBuilder", bc.new_(StringBuilder.class)));
+    }
+
+    /**
+     * Construct a new instance on a newly created {@link StringBuilder}
+     * with given {@code capacity}. The {@code StringBuilder} instance
+     * may be obtained using {@link #getInstance()}.
+     *
+     * @param bc the block creator (must not be {@code null})
+     * @param capacity the capacity of the newly created {@link StringBuilder}
+     */
+    public StringBuilderOps(final BlockCreator bc, final int capacity) {
+        super(StringBuilder.class, bc, bc.new_(StringBuilder.class, Constant.of(capacity)));
+    }
+
+    public StringBuilderOps append(final Expr expr) {
```

Comment:
The rest of these methods should have javadoc as well.

Reply:
Ah yeah, good point. Will add.

Reply:
Done.